cs-commons/src/main/java/io/cloudslang/content/constants/InputNames.java

```diff
@@ -0,0 +1,7 @@
+package io.cloudslang.content.constants;
+
+/**
+ * Created by victor on 31.08.2016.
+ */
+public class InputNames {
```

Comment:
to avoid so many classes maybe add a ContractUtils that has 4 static nested class `InputNames`, `OutputNames `and so on